Sources/TokamakCore/MountedViews/MountedElement.swift

```diff
@@ -98,24 +98,29 @@ public class MountedElement<R: Renderer> {
     }
   }
 
+  public internal(set) var viewTraits: _ViewTraitStore
```

Comment:
Are these traits somehow different from SwiftUI accessibility traits? I can't find anything else about SwiftUI traits, just wondering what's used as a reference here.

Reply:
SwiftUI has `_ViewTraitKey` internally, and uses them for `transition`, `onDelete`, `onMove` and some other modifiers (I think drag & drop?). From what I can tell, a trait is data attached to a single view, and in many cases seems to be a way to get the trait out of SwiftUI and into the host platform (for instance, for UITableView actions or NSItemProvider-based drag & drop).

Although I'm really just speculating as to how SwiftUI uses them by looking at the `swiftinterface`. In Tokamak they are meant to be passed to the nearest host element so renderers can access them.

Reply:
Traits are also used for `zIndex(_:)` and `layoutPriority(_:)`. I haven't look into whether Tokamak has implemented these modifiers. If they're not implemented, traits would be really helpful.